man/pam_systemd.xml

```diff
@@ -259,6 +259,51 @@
     </variablelist>
   </refsect1>
 
+  <refsect1>
+    <title>Session limits</title>
+
+    <para>The following pam environment variables are read by the module
```

Comment:
I think we should change this to "PAM", acronyms are generally uppercased after all